client/src/containers/projects/url-store.ts

```diff
@@ -1,19 +1,21 @@
-import {
-  COST_TYPE_SELECTOR,
-  PROJECT_PRICE_TYPE,
-} from "@shared/entities/projects.entity";
+import { COST_TYPE_SELECTOR } from "@shared/entities/projects.entity";
 import { parseAsJson, useQueryState } from "nuqs";
 import { z } from "zod";
 
 import { FILTER_KEYS } from "@/app/(overview)/constants";
 
+export enum CUSTOM_PROJECT_PRICE_TYPE {
+  INITIAL_CARBON_PRICE_ASSUMPTION = "Initial carbon price assumption",
+  BREAKEVEN_PRICE = "Breakeven price",
+}
+
```

Comment:
@atrincas Being picky here, but I would put this in the custom-project folder. It's weird if I have to do 
`import { CUSTOM_PROJECT_PRICE_TYPE } from @/containers/projects/url-store`.